app/controllers/api/v0/versions_controller.rb

```diff
@@ -13,6 +13,49 @@ def index
     }
   end
 
+  def sampled
+    @sampling = true
+    raise API::NotFoundError('You must provide a page to sample versions of.') unless page
+
+    # TODO: support variable sample periods. Need to figure out a reference
+    # point for when those periods start.
+    # We don't support the complex filters and options #index does; we want to
+    # keep this as simple and fast as possible.
+    query = page.versions
+    # FIXME: this should probably have special pagination by date, since we
+    # don't want a sample group split across two responses.
+    paging = pagination(query)
```

Comment:
May not want to wait for this, since it will be messy, and for the endpoint to be useful we also need to call it from the -ui project. Probably not worth blocking that work on this.